android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -195,11 +190,7 @@ fun Context.copyImage(
     // Only show a Snackbar for Android 12 and lower.
     // See developer.android.com/develop/ui/views/touch-and-input/copy-paste .
     if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.S_V2) {
```

Comment:
If we're keeping this version check and comment here, then we should document that in the `param` for `onCopyConfirmation` and update the comment too.

The alternative is to also lift this check to the caller (which is probably preferable?).

Reply:
Good suggestion on lifting this check 👍